src/core/fonts.js

```diff
@@ -834,6 +834,7 @@ var Font = (function FontClosure() {
       if ((usedFontCharCodes[fontCharCode] !== undefined ||
            isProblematicUnicodeLocation(fontCharCode) ||
            (isSymbolic && !hasUnicodeValue)) &&
+          !missingGlyphs[glyphId] &&
```

Comment:
Just asking, and it might be a totally pointless micro-optimization, but would it make sense (at all) to check `!missingGlyphs[glyphId]` *first* in this condition? I.e. instead do:

```javascript
if (!missingGlyphs[glyphId] &&
    (usedFontCharCodes[fontCharCode] !== undefined ||
     isProblematicUnicodeLocation(fontCharCode) ||
     (isSymbolic && !hasUnicodeValue)) &&
    nextAvailableFontCharCode <= PRIVATE_USE_OFFSET_END) { // Room left.
```

The reason that I'm asking is that we could then avoid pointless `isProblematicUnicodeLocation` function calls when we *know* that a `glyph` is missing, since I'm assuming that it's cheaper to lookup `missingGlyphs[glyphId]` than it's calling a function?

Again, this might a completely unnecessary change, but I figured it can't hurt to ask!